factory/tools/manual_glidein_submit.py

```diff
@@ -152,7 +152,8 @@ def main():
             ad_gc, frontend_descript, glidein_descript.data['PubKeyObj'])
         security_class = sym_key_obj.decrypt_hex(
             ad_gc['GlideinEncParamSecurityClass']) # GlideinSecurityClass
-        proxyid = sym_key_obj.decrypt_hex(ad_gc['GlideinEncParamSubmitProxy'])
+        proxyid = sym_key_obj.decrypt_hex(ad_gc['GlideinEncParamSubmitProxy']).decode('utf8')
+        security_class = security_class.decode('utf8')
```

Comment:
You could add `.decode('utf8')` at the end of line 154, to be consistent w/ proxyid

Reply:
Done